sql/databases.go

```diff
@@ -110,8 +110,8 @@ type ReadOnlyDatabase interface {
 // TableCreator is a Database that can create new tables.
 type TableCreator interface {
 	Database
-	// CreateTable creates the table with the given name and schema.
-	CreateTable(ctx *Context, name string, schema PrimaryKeySchema, collation CollationID) error
+	// CreateTable creates the table with the given name, schema, collation, and comment.
```

Comment:
Maybe leave a comment here about what an integrator must do for the comment to be used by the engine after this. There's a weird asymmetry where you are required to accept a comment at creation time but not required to return it.